datastore/index.yaml

```diff
@@ -0,0 +1,54 @@
+indexes:
+
+- kind: Task
+  ancestor: no
+  properties:
+  - name: done
+  - name: priority
+    direction: desc
+
+- kind: Task
+  properties:
+  - name: collaborators
+    direction: asc
+  - name: created
+    direction: desc
+
+- kind: Task
+  ancestor: no
+  properties:
+  - name: priority
+    direction: asc
+  - name: created
+    direction: asc
+
+- kind: Task
+  ancestor: no
+  properties:
+  - name: done
+  - name: priority
+    direction: desc
+  - name: created
+    direction: asc
+
```

Comment:
@remi Fantastic. I'm not sure how I created that index in my own env and then missed adding that index to the file.